### PR TITLE
Generate documentation using go templates

### PR DESCRIPTION
**Pull request by zregvart**
Reference: EC-513

### CODE REVIEW

**Comment by simonbaird** [comment]
Didn't test it, but I looked the commits - seems alright to me.

File: docs/modules/ROOT/pages/ec_purl_parse.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+== Return
+
+`object` (`object`): the parsed PURL object
+
+The object contains the following attributes:
+
+* `name` (`string`)
+* `namespace` (`string`)
+* `qualifiers`(`array`)
+**[object<key: string, value: string>]
```
Comment (lcarva) [inline docs/modules/ROOT/pages/ec_purl_parse.adoc:22]: Does this need a space after `**` ?

File: tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml [hunk 2]
```diff
@@ -83,18 +87,18 @@ spec:
...
         certificates issued by a not-commonly trusted root CA. In such cases,
-        "/var/run/secrets/kubernetes.io/serviceaccount" is a good value. Multiple
-        paths can be provided by using the ":" separator.
+        `/var/run/secrets/kubernetes.io/serviceaccount` is a good value. Multiple
+        paths can be provided by using the `:` separator.
       default: ""
 
     - name: INFO
       type: string
-      description: Include rule titles and descriptions in the output. Set to "false" to disable it.
```
Comment (lcarva) [inline tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml:92]: I think it should be `"false"`. The value has to be a string.

**Review by lcarva — APPROVED** [review]
Sweet!

File: docs/modules/ROOT/pages/ec_purl_parse.adoc (hunk 1 above)
Comment (zregvart) [reply docs/modules/ROOT/pages/ec_purl_parse.adoc:22]: It does, good catch

**Pull request merged**